### ISSUE TITLE
On the book list, "Categories" and "Genres" might end up being redundant. Guess we could wait and see how categories develop though.

### ISSUE THREAD

**Issue by jzthompson** [maltyeva/mindimp-pg#6]

**Comment by maltyeva** [comment]
Yes, I definitely agree. I'm trying to figure out the best way to organize categories, because they can serve a multi-purpose of both being a category (e.g British Lit), a certification (which can also be say, British Lit, or French Romanticism), or a Course Syllabus (Western Cannon Spring 2015), while genres are limited to being genres, but yeah, let's see how we use it and we can adjust either of the fields.

**Comment by maltyeva** [comment]
Hey Jeff, 

I've been giving a little more thought to genres and categories and I thought the Genres could be replaced by something like "Keywords", which we might use to make the books more searchable. What  do you think of that?

**Comment by jzthompson** [comment]
Yeah, good idea! Keywords would be good to have for books too.

**Comment by maltyeva** [comment]
great! I'll work on implementing it and keep you updated
